democracy/src/lib.rs

```diff
@@ -495,14 +497,14 @@ pub mod pallet {
 					<encointer_communities::Pallet<T>>::do_update_demurrage(cid, demurrage)?;
 				},
 				ProposalAction::UpdateNominalIncome(cid, nominal_income) => {
-					<encointer_communities::Pallet<T>>::do_update_nominal_income(
+					<pallet_encointer_communities::Pallet<T>>::do_update_nominal_income(
 						cid,
 						nominal_income,
 					)?;
 				},
 
 				ProposalAction::SetInactivityTimeout(inactivity_timeout) => {
-					<encointer_ceremonies::Pallet<T>>::do_set_inactivity_timeout(
+					<pallet_. encointer_ceremonies::Pallet<T>>::do_set_inactivity_timeout(
```

Comment:
dafuq, how could this be merged? @brenzi, @pifragile ?

Reply:
you somehow commented on an outdated version of the diff.
<img width="795" alt="image" src="https://github.com/encointer/pallets/assets/14249275/2ce508d2-080e-44a0-993b-ee8eb755f36b">

see here:
https://github.com/encointer/pallets/pull/362/files#diff-52647a2848a1f05fd742a15c942a8f46773d88eaae8c8fee81f3cc90ad99f90dR517

Reply:
Ah, ok weird, don't know how I ended up there. XD